.buildkite/toml/bomex.toml

```diff
@@ -0,0 +1,79 @@
+[config] 
+value = "column" 
```

Comment:
Could we add the toml files for the other SCM EDMF test cases as well? I would like to keep them all in the same format

Reply:
Or alternatively, we could just  do it for one case (Bomex) and keep the others as they are for now. And even add a CI job that tests if the `toml` driven and `cli_args` driven simulations are the same

Reply:
Will do.

Reply:
Sounds good to me. We can have Bomex and toml driven Bomex